File: Day 5/5th-day.java
/*
Task
i = result.
Given an integer, n. print its first 10 multiples. Each multiple n x i (where 1 ≤ i ≤ 10) should be printed on a new line in the form: n x i = result
*/


import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Solution {
    public static void main(String[] args) throws IOException {
        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));

        String nTemp = bufferedReader.readLine().trim();

        int n = Integer.parseInt(nTemp);

        printMultiplicationTable(n);

        bufferedReader.close();
    }

    public static void printMultiplicationTable(int n) {
        for (int i = 1; i <= 10; i++) {
            int result = n * i;
            System.out.println(n + " x " + i + " = " + result);
        }
    }
}

File: Day 5/5th-day.php
<!-- 
Task
i = result.
Given an integer, n. print its first 10 multiples. Each multiple n x i (where 1 ≤ i ≤ 10) should be printed on a new line in the form: n x i = result
 -->


 <?php
$n = intval(trim(fgets(STDIN)));

printMultiplicationTable($n);

function printMultiplicationTable($n) {
    for ($i = 1; $i <= 10; $i++) {
        $result = $n * $i;
        echo "$n x $i = $result\n";
    }
}
?>
